.devfile.yaml

```
@@ -75,6 +75,15 @@ commands:
         oc project $(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace) && \
         mvn -pl northwind oc:remote-dev
       workingDir: ${PROJECT_SOURCE}
+  - id: cleanup
+    exec:
+      label: "6. Clean Up"
+      component: tools
+      commandLine: |
+        source /home/user/.bashrc && \
+        oc project $(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace) && \
+        oc delete all --selector app.kubernetes.io/part-of=northwind-traders
+      workingDir: ${PROJECT_SOURCE}
```

```suggestion
        oc delete all --selector 'group in (northwind-traders,northwind-industries)'
      workingDir: ${PROJECT_SOURCE}
```

I changed the query as I still had a few routes, services etc...
I have removed some lines as you don't need them for this command.